test/Optimizer/rlexe.xml

```diff
@@ -1485,4 +1485,12 @@
       <compile-flags>-maxinterpretcount:1 -maxsimplejitruncount:1 -oopjit- -off:bailonnoprofile -loopinterpretcount:0 -OOPJITMissingOpts- </compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>mul_rejit_bug.js</files>
+      <baseline>mul_rejit_bug.baseline</baseline>
+      <compile-flags>-mic:1 -oopjit- -bgJit- -off:simplejit -trace:bailout</compile-flags>
```

Comment:
I had to use `trace:bailout` here. `testtrace:rejit` does not seem to report jit-time bailouts.

Reply:
BTW, `rejit` trace seems to use 'CR' (mac) line ending and the output differ cares about the difference. Not sure if that is intentional.

Reply:
So I checked the switches that we have and I think I would prefer if you used `-trace:rejit` after all.
Because we try to avoid having bytecode offsets in baseline files as much as possible (e.g.: `exponent ( (#1.2), #3) offset: #0016`). Those are really annoying because we have to update baselines whenever something in the bytecode changes.
Also, what you are really trying to check is to make sure we don't have an extra rejit at jit time.

For the line-ending issue. You can do is the following to resolve it.
Inside the `test` folder run ` runtests.cmd -x64debug -dirs Optimizer -rebase`
`-dirs Optimizer` is to run the tests only in the Optimizer folder
`-rebase` will generate `test/Optimizer/mul_rejit_bug.baseline.rebase` which you can then use to overwrite the original baseline file and then our test runner should be happy with the line endings

Reply:
Right. Offsets in baseline can be a problem.

I will try `trace:rejit` again. Last time I tried it did not appear to be sensitive to JIT-time bailouts, but I could be doing something incorrectly. 
And there was that line ending confusion - thanks for the tip on how to deal with that without involving an editor.

Reply:
Looks like I was wrong - `trace:rejit` does report jit time bailouts and _is_ sensitive to the presence of the fix.

Reply:
Actually. It seems that `testtrace:rejit` does not catch the presence of the fix and that was the one I tried initially, while `trace:rejit` does. 

Anyways, will use `trace:rejit` now.